src/WebSocket.cs

```diff
@@ -66,10 +65,6 @@ public async Task Connect(string? auth, string host, string nameOrAddress, Conne
             {
                 Ws.Options.SetRequestHeader("Authorization", $"Bearer {auth}");
```

Comment:
I ran into issues here as well, I don't think `browser` supports `SetRequestHeader`.

You can use a param in the querystring instead though like
```
if (auth != null)
{
   uri += $"&token={Uri.EscapeDataString(auth)}";
}
```